File: ClientApp/src/types/enums/Rola.ts
export enum Rola {
    Student = "Student",
    Asistent = "Asistent",
    Profesor = "Profesor",
    Administrator = "Administrator"
}

export const RoleColors = new Map<string, string>([
    [Rola.Student, 'green'],
    [Rola.Asistent, 'geekblue'],
    [Rola.Profesor, 'volcano'],
    [Rola.Administrator, 'red'],
]);

File: Models/Enums/Rola.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Models.Enum
{
    public enum Rola
    {
        [Description("Student")]
        Student = 0,
        [Description("Asistent")]
        Asistent = 1,
        [Description("Profesor")]
        Profesor = 2,
        [Description("Administrator")]
        Administrator = 3
    }
}
